.github/workflows/build.yml

```diff
@@ -6,9 +6,12 @@ on:
     branches:
       - master
 env:
-  APP_REPOSITORY:  dfedigital/get-teacher-training-adviser-service
-  PROJECT:         Teacher Training Adviser Service
-  DOMAIN:          london.cloudapps.digital
+  DOCKERHUB_REPOSITORY: dfedigital/get-teacher-training-adviser-service
+  DOMAIN:               london.cloudapps.digital
+  PROJECT:              Teacher Training Adviser Service
+  REVIEW_APPLICATION:   review-teacher-training-adviser
+  APPLICATION_NAME:     get-teacher-training-adviser-service
```

Comment:
If you rename the above, this could become `paas_application_name` or `paas_app_name`